queries.sql

```diff
@@ -90,4 +90,72 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals
  LEFT JOIN animals ON owners.id = animals.owner_id
  GROUP BY owners.id
  ORDER BY num_animals DESC
+LIMIT 1;
+
+SELECT a.name AS animal_name, v.visit_date
+ FROM visits v
+ JOIN animals a ON v.animal_id = a.id
+ JOIN vets vt ON v.vet_id = vt.id
+ WHERE vt.name = 'William Tatcher'
+ ORDER BY v.visit_date DESC
+LIMIT 1;
+
+SELECT COUNT(DISTINCT animal_id)
+ FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+SELECT v.name AS vet_name, s.name AS specialty_name 
+FROM vets AS v
+LEFT JOIN specializations AS sp ON v.id = sp.vet_id
+LEFT JOIN species AS s ON sp.species_id = s.id
+ORDER BY v.name ASC;
+
+SELECT a.name as animal_name, v.name as vet_name, vs.name as species_name, vi.visit_date 
+FROM visits vi
+JOIN animals a ON vi.animal_id = a.id
+JOIN vets v ON vi.vet_id = v.id
+JOIN specializations sp ON sp.vet_id = v.id
+JOIN species vs ON sp.species_id = vs.id
+WHERE v.name = 'Stephanie Mendez' AND vi.visit_date BETWEEN '2020-04-01' AND '2020-08-30'
+ORDER BY vi.visit_date;
```

Comment:
- [ ] Let us kindly fix the repetition of the query result numbered 4 in the screenshot `List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020?`

Reply:
![Changes 1](https://user-images.githubusercontent.com/109960845/236485528-79e9779e-4514-4044-acef-b8e9ea25cf51.PNG)
Repetition of query fixed✔